packages/salesforcedx-utils-vscode/src/test/orgUtils.ts

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { CliCommandExecutor, CommandOutput, SfdxCommandBuilder } from '../cli';
+import childProcess = require('child_process');
+import * as path from 'path';
+import * as util from 'util';
+
+// Used only for CI purposes. Must call delete if you call create
+export async function createSFDXProject(projectName: string): Promise<void> {
+  const execution = new CliCommandExecutor(
+    new SfdxCommandBuilder()
+      .withArg('force:project:create')
+      .withFlag('--projectname', projectName)
+      .withJson()
+      .build(),
+    { cwd: process.cwd() }
+  ).execute();
+  const cmdOutput = new CommandOutput();
+  await cmdOutput.getCmdResult(execution);
+  return Promise.resolve();
+}
+
+export async function createScratchOrg(projectName: string): Promise<string> {
+  const scratchDefFilePath = path.join(
+    process.cwd(),
+    projectName,
+    'config',
+    'project-scratch-def.json'
+  );
+  const execution = new CliCommandExecutor(
+    new SfdxCommandBuilder()
+      .withArg('force:org:create')
+      .withFlag('--definitionfile', `${scratchDefFilePath}`)
+      .withArg('--setdefaultusername')
```

Comment:
@Sweetman I added defaultusername here.

Reply:
@chowwinston was there a reason for creating the scratch org related utils you created for your sobject tests in `salesforcedx-sobjects-faux-generator`? René and I were discussing this and would prefer to move this into the utils package but would like to double check with you.

Reply:
I just kept it in my package to get sobject stuff up and running first. Utils is the better place to put it and thanks for migrating this over @muenzpraeger. I can refactor to point to this util after it gets merged.